frontend/package.json

```diff
@@ -26,9 +26,9 @@
     "e2e-employee-2": "testcafe 'chrome --start-fullscreen' src/e2e-test/specs/2_employee-2/*.spec.ts",
     "e2e-mobile": "testcafe 'chrome --start-fullscreen' src/e2e-test/specs/6_mobile/*.spec.ts",
     "e2e-messaging": "testcafe 'chrome --start-fullscreen' src/e2e-test/specs/7_messaging/*.spec.ts",
-    "e2e-playwright": "jest --testTimeout 60000 --runInBand src/e2e-playwright/specs",
-    "e2e-ci-testcafe": "BASE_URL=http://localhost:9999 testcafe 'chrome:headless:emulation:width=1920;height=1080' -s takeOnFails=true --video videos --video-options failedOnly=true --stop-on-first-fail",
-    "e2e-ci-playwright": "BASE_URL=http://localhost:9999 jest --testTimeout 60000 --runInBand --bail 1"
+    "e2e-playwright": "LANG=fi-FI LANGUAGE=fi_FI jest --testTimeout 60000 --runInBand src/e2e-playwright/specs",
```

Comment:
Why the `LANG` and `LANGUAGE` envs?

Reply:
Noticed that if user has different LANGUAGE or LANG environment variables set then those are passed to chromium and then tests fail.

Reply:
Weird, 'cause I've always used `LANG=en_US.UTF-8` locally and never had those problems. And besides, I think we should actually then fix those tests to either a) not make assumptions about user language and rely on `data-qa` labels, or b) explicitly select a language in eVaka.

Did this break in CI, too, without the variables?

Reply:
CI uses LANG: https://github.com/espoon-voltti/evaka/blob/master/.circleci/config.yml#L462

Why I added those here was because I spend some time to find out why tests were failing locally but not in CI. Also noticed that you need overwrite both of those if those are defined. That LANG requirement has not been long.